_posts/blog/http/2018-11-08-napetitions-review.md

```diff
@@ -0,0 +1,51 @@
+
+**Significance**  
+ The methodological priorities (tagging, keywords, citation, etc.) place signers and the groups involved in petitioning efforts at the forefront of the site. In both metadata and archivist notes, The Native American Petitions Dataverse highlights indigenous voices, allowing users to filter petitions by keyword term, publication year, production date (year petition was signed/submitted), and geographic coverage. This reveals opportunities to restructure classic periodization and overemphasized themes in the historiography. More significantly, chosen methods of tagging and consistent categorization increase the visibility of traditionally disenfranchised figures. I am thinking particularly about Native women’s participation in petitioning campaigns and their advocacy about issues related to both individual and tribal rights.  
+
+Take, for example, the 1762 petition of Esther Anthony, a Native woman of color from the tribe of Mashpee, but living in the town of Middleborough, Massachusetts. The [informational page](https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/W0R4MJ) includes several identifiers, including the total number of signatures, the extent of legislative action, the number of females of color signatures, tribal affiliation, and additional archivist notes (i.e., widow, Jeremiah Anthony, sale, lands, payments). At the top of the page is a full citation of the original primary source along with the petition subject (which, in this case is property) and a link to a [high resolution photograph](https://iiif.lib.harvard.edu/manifests/view/drs:422116544$140i) of the petition. In the Harvard Library viewer, users can "flip" through the selected petition or browse related petitions (left of the screen) either submitted on the same date, on similar topics, or by the same petitioners as they would be organized in an archival folder. This user-friendly layout provides ample room for researchers interested in larger historical trends, genealogical networks, or public history projects. The latter opens the door to tellings of more nuanced narratives, powerful stories, specific individuals names, and the movement of ideas. Dr. Carpenter demonstrates one such application in a data visualization project called [“Your Humble Petitioners”.](http://www.yourhumblepetitioners.com/mapview/antislaveryma) This interactive map associates geographic coordinates with the locations inscribed in the metadata of the antislavery petitions. This could very easily be replicated with the Native American petitions. 
```

Comment:
The phrasing around "Your Humble Petitioners" is a bit confusing because it sounds like it's part of this collection, when it's actually from a different project. Please clarify.